app/code/Magento/Developer/Console/Command/DiInfoCommand.php

```diff
@@ -173,4 +194,40 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         return \Magento\Framework\Console\Cli::RETURN_SUCCESS;
     }
+
+    /**
+     * Set Area for DI Configuration
+     *
+     * @param string $area
+     * @return void
+     * @throws \InvalidArgumentException
+     */
+    private function setDiArea($area)
```

Comment:
Please specify the return type of the method:

for e.g.
```suggestion
    private function setDiArea($area): void
```